src/root/utils/field-report-constants.js

```diff
@@ -54,6 +54,11 @@ export const countries = (countries) => {
     countriesSelectList = countries.map(country => ({ value: country.id, label: country.name }));
   }
 
+  // show only independent countries.
+  countriesSelectList = countriesSelectList.filter(country => {
```

Comment:
This won't work in the case of:
https://github.com/IFRCGo/go-frontend/blob/edd6de367a66b7eae24ea2757ce161e0bedbcdfd/src/root/utils/field-report-constants.js#L53-L55

For example it doesn't work for me on the Registration page or the Create New Field Report.

Just a suggestion (doesn't need to change), but something like this would be a bit more clear/n for me:
```javascript
countriesSelectList = countriesSelectList.filter(country => country.independent === true);
```